tasmota/xdrv_29_deepsleep.ino

```diff
@@ -154,9 +154,13 @@ void DeepSleepStart(void)
 }
 
 void DeepSleepEverySecond(void)
-{
+{  
+  //AddLog(LOG_LEVEL_DEBUG_MORE, PSTR("Wifi Info: up %d, wifidown %d, wifistatus %d, flag %d"),TasmotaGlobal.uptime, TasmotaGlobal.global_state.wifi_down, Wifi.status , deepsleep_flag);
+  if (TasmotaGlobal.uptime > 15 && Wifi.status != WL_CONNECTED && !deepsleep_flag && DeepSleepEnabled()) {
```

Comment:
@stefanbode Could you please make this value with a #define that can be overridable from user_config_override ?
I see cases where connection with the router is difficult on an outdoor sensor and being able to adjust that time could be usefull.
Thanks

Reply:
Yes, makes absolute sense. You can now define in seconds or totally disable with "0".

Reply:
Are you fine with the default? Or should we increase it a bit?

Reply:
I think it's fine.

I suggest to put the define between 
```
#ifndef ...

#endif 
```
so users can change the default from the user_config.h file.

Reply:
yep, done

Reply:
@arendst : Again you're absolutely right. Did not work the way I did it. Now it should be ok. Thanks for the hint.

Reply:
Many thanks @stefanbode